content/en/docs/instrumentation/python/mypy.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Using mypy
+weight: 5
+---
+
+If you're using [mypy](http://mypy-lang.org/), you'll need to turn on [namespace
+packages](https://mypy.readthedocs.io/en/stable/command_line.html#cmdoption-mypy-namespace-packages).
```

Comment:
```suggestion
If you're using [mypy](http://mypy-lang.org/), you'll need to turn on [namespace
packages](https://mypy.readthedocs.io/en/stable/command_line.html#cmdoption-mypy-namespace-packages), otherwise `mypy` won't be able to run correctly.
```